src/main/java/xyz/deseteral/lebkuchenfm/services/commands/RootCommandProcessor.java

```diff
@@ -0,0 +1,25 @@
+package xyz.deseteral.lebkuchenfm.services.commands;
+
+import org.springframework.stereotype.Component;
+import xyz.deseteral.lebkuchenfm.domain.Command;
+import xyz.deseteral.lebkuchenfm.domain.CommandProcessingResponse;
+
+import java.util.List;
+
+@Component
+public class RootCommandProcessor {
```

Comment:
Musi być publiczne? Może jest to kawałek domeny?

Reply:
Fixed